Dockerfile-s390x

```diff
@@ -0,0 +1,55 @@
+# Copyright 2015 Tigera, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# For details and docs - see https://github.com/phusion/baseimage-docker#getting_started
+
+# The Dockerfile-s390x is copied from routereflector/Dockerfile.
+# Modifications done includes:
+# 1) Base image has been changed from ubuntu:14.04 to s390x/ubuntu:16.04
+# 2) Added python3-minimal dependency
+
+FROM s390x/ubuntu:16.04
+
+RUN apt-get update && \
+        apt-get install -y python3-minimal
```

Comment:
Please can you add `&& rm -rf /var/lib/apt/lists/*` to this command.  It leaves a cleaner/smaller image.

Reply:
@fasaxc, we will incorporate said change. Thanks.